src/plugins/discover/public/__mocks__/ui_settings.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { IUiSettingsClient } from 'kibana/public';
+import { SAMPLE_SIZE_SETTING } from '../../common';
+
+export const uiSettingsMock = ({
+  get: (key: string) => {
+    if (key === SAMPLE_SIZE_SETTING) {
+      return 10;
+    }
+    return false;
```

Comment:
Why false here? Can we return 0 or something for consistency?

Reply:
It was copy and past of the data grid PR, can also improved there. Good catche, I've removed the return. we will need this more often in our unit tests, and can extend it this with valid returns.